libs/blocks/mobile-app-banner/mobile-app-banner.js

```diff
@@ -0,0 +1,73 @@
+import { getConfig } from '../../utils/utils.js';
+
+async function getKey(product) {
+  const config = getConfig();
+  let keyMatch = [];
+  const resp = await fetch(`${config.contentRoot ?? ''}/branch-io-key.json`);
+  if (resp.ok) {
+    const json = await resp.json();
+    keyMatch = json.data.filter(
+      (p) => p.product === product,
+    );
+  }
+  return keyMatch[0]?.key;
```

Comment:
```suggestion
  return json.data.find(p => p.product === product)?.key;
```

Reply:
Thanks for the review @mokimo . I had tried below code initially
return json.data.find((p) => p.product === product)?.key;

However as I was getting lint error hence went with the current approach.